src/main/webapp/i18n/de/userSettings.json

```
@@ -27,6 +28,20 @@
                 "sshKeyDisplayedInformation": "Das ist dein aktuell konfigurierter SSH-Schlüssel:",
                 "key": "SSH Schlüssel"
             },
+            "vcsAccessTokensSettingsPage": {
+                "infoText": "Du kannst ein persönliches Zugriffstoken generieren, um mit der Artemis Local Version Control System zu interagieren. Verwende es um dich über HTTP bei Git zu authentifizieren.",
+                "deleteVcsAccessTokenQuestion": "Möchtest du dein Zugriffstoken für das Versionskontrollsystem wirklich löschen? Du kannst dich nicht mehr bei Repositories authentifizieren, die mit diesem Token geklont wurden.",
```

```suggestion
                "deleteVcsAccessTokenQuestion": "Möchtest du dein Zugriffstoken für das Versionskontrollsystem wirklich löschen? Du kannst dich nicht mehr bei lokalen Repositories authentifizieren, die mit diesem Token geklont wurden.",
```
Otherwise someone might think that they can no longer interact with the remote repository